generate/main.go

```diff
@@ -57,11 +57,11 @@ func writeMappingToFile(f *os.File, m map[string]string, mapName string) {
 	}
 	printf("package %s\n\n", pkgName)
 	printf("// Code generated by generate/main.go DO NOT EDIT\n")
-	printf("var %s = map[string]string {", mapName)
+	printf("var %s = [...]struct{ cidr, asn string }{", listName)
 	for k, v := range m {
-		printf("\n\t \"%s\": \"%s\",", k, v)
+		printf("\n\t{\"%s\", \"%s\"},", k, v)
 	}
-	printf("\n}")
+	printf("}\n")
```

Comment:
do you want a `\n` both in front and after this final closing brace?

Reply:
Yeah you are right. 
`go fmt` fixed it anyway.

Reply:
@willscott fixed:
```diff
-	{"2a02:6300::/42", "50128"}}
+	{"2a02:6300::/42", "50128"},
+}
```